python/nammu/view/ToolbarView.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, controller):
         tooltips['redo'] = 'Redo last undone action'
         tooltips['copy'] = 'Copy text selection'
         tooltips['cut'] = 'Cut text selection'
-        tooltips['validate'] = 'Check current ATF correctness'
         tooltips['paste'] = 'Paste clipboard content'
+        tooltips['validate'] = 'Check current ATF correctness'
```

Comment:
This change shouldn't be in this branch, but I didn't realise. It fixes the ordering of one button in the toolbar and conflicts with the development branch.